### PR TITLE
docs(chinese):mirror

### PR DESCRIPTION
**Pull request by shaonianche**
Signed-off-by: shaoanianche <github@duanfei.org>

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies:
   markdown-it-footnote: 3.0.3
 
 devDependencies:
-  vitepress: 1.0.0-alpha.9_@algolia+client-search@4.13.1
+  vitepress: 1.0.0-alpha.9_ygzyjl425qrcgm3zeayiliwge4
```
Comment (gaocegege) [inline pnpm-lock.yaml:14]: Please remove these changes in the lock yaml file.

Comment (gaocegege) [reply pnpm-lock.yaml:14]: Or @Sepush could you please help review it?

**Review by gaocegege — COMMENTED** [review]
Generally LGTM with some nits.

**Review by Sepush — APPROVED** [review]

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

**Pull request merged**